### DISCUSSION TITLE
Error: PrismaClient is unable to be run in the browser.

### DISCUSSION THREAD

**Discussion by rajanshresth** [prisma/prisma#19239] [Q&A] [answered]
Latest version of nextjs(with app dir) and prisma.

![image](https://github.com/prisma/prisma/assets/76089000/a3c84b35-56c4-47ac-9047-f62575a871f9)

**Comment by ludralph** [comment, marked as answer]
Hi @rajanshresth 👋 

Can you please take a look at this [issue](https://github.com/prisma/prisma/issues/6219). It has some solutions to this same problem. Also, you might want to look at this [documentation](https://www.prisma.io/docs/guides/other/troubleshooting-orm/help-articles/nextjs-prisma-client-dev-practices) for instantiating Prisma Client with Next.js

**Comment by rajanshresth** [comment]
Thanks for the response!!!
